v3/README.md

```diff
@@ -70,6 +70,23 @@ func main() {
 }	
 ```
 
+### Findable
+
+Most of the list request `ListX()` return a type containing the list of the resource requested and a method `FindX()` to be able to retrieve a resource by its `name` or `id` most of the time.
```

Comment:
`X` in `FindX()` is not really needed. All functions could just be named `Find()` as they are method of specific struct.
It's a breaking change though, you can decide if it's worth changing or leave it as is.

Reply:
For the moment we will leave as it is to not breaking change....